feature-group-definitions/edit-context.yml

```
@@ -1 +1,2 @@
+name: "Edit Context"
```

To match the API name:

```suggestion
name: "EditContext"
```